Security (Is it safe?)/Build Quality/Best Build Practices.yaml

```diff
@@ -56,4 +56,12 @@ criterias:
 
             How complex is the code?
 
+      - indicator: |+
+            The software was built reproducibly.
+
+        procedures:
+          - |+
+            If the project is freeware or open source, build the project and verify the output.
```

Comment:
Do you mean "Free Software" instead of "freeware"? Because the latter can come without source and just comes with the freedom to distribute and run.